packages/@okta/vuepress-site/code/react/okta_react/index.md

```diff
@@ -442,14 +443,14 @@ export default withRouter(class AppWithRouterAccess extends Component {
 
   render() {
     return (
-      <Security issuer='https://${yourOktaDomain}/oauth2/default'
+      <Security issuer='https://{yourOktaDomain}/oauth2/default'
```

Comment:
Same comment here about switch and multiple components rendering on a single route

Reply:
Used `history.push('/login')` to solve.